src/include/expression/tuple_value_expression.h

```diff
@@ -94,6 +94,8 @@ class TupleValueExpression : public AbstractExpression {
  protected:
   TupleValueExpression(const TupleValueExpression &other)
       : AbstractExpression(other),
+        is_bound(other.is_bound),
+        bound_obj_id(other.bound_obj_id),
```

Comment:
The name of member variables of a class should end with an additional underscore.